Makefile

```diff
@@ -94,10 +94,6 @@ install-npm:
 check-markdown: install-npm
 	npm run check-markdown
 
-.PHONY: format-markdown
-format-markdown: install-npm
-	npm run format-markdown
-
```

Comment:
I'd rather keep it and fix appropriately. See WIP PR https://github.com/stac-extensions/mlm/pull/83

Reply:
ok thanks!